roles/confluent.test/tasks/check_user.yml

```diff
@@ -0,0 +1,13 @@
+---
+    - name: Check user on service
```

Comment:
This test appears to check if a portion of a user's full name (ie c3-cust+) owns any process. Is there any way to be more specific that its the c3 process as well as use the full name?

Reply:
There is only one process on the host that could be running with that username, so not sure why we would need to be more specific.  The check of the name is what is show when you grep for processes.  There are other ways to get this info but they are more complicated and would require stripping of spaces or null values.  I investigated this already and thought this was the simplest approach.